ubuntu/debian/control

```diff
@@ -7,9 +7,10 @@ Build-Depends: cmake,
                doxygen,
                pkg-config,
                debhelper (>= 9),
-               libignition-cmake2-dev,
+               libignition-cmake2-dev (>= 2.3.0),
                libignition-common3-core-dev (>= 3.0.0),
                libignition-msgs6-dev,
+               libignition-tools-dev,
```

Comment:
so this is adding it as a build depend. is that what you want? I think having this at build time just enables an extra test; I believe it doesn't change what is installed

Reply:
Ok, that's probably not what I want. But I just copied it from `ign-msgs`:

https://github.com/ignition-release/ign-msgs6-release/blob/bccc4fd8f3b51d15baab7db95234f8051a76d83b/ubuntu/debian/control#L15

Should I move it to `Depends` below?

Reply:
It's also `Build-Depends` on `ign-transport`: https://github.com/ignition-release/ign-transport9-release/blob/238b8964ef0affd4bcbd65126559a9313b20c4ca/ubuntu/debian/control#L16

Maybe @j-rivero can shed a light?

Reply:
if your goal is that `libignition-tools-dev` is installed whenever someone installs `libignition-fuel-tools5-dev`, then yes, move it to `Depends` below

Reply:
How about the current state:

* `ignition-fuel-tools5` installs `ignition-tools`, with `Depends`
* `libignition-fuel-tools5-dev` `Build-depends` on `libignition-tools-dev`

Reply:
> Maybe @j-rivero can shed a light?

I can probably give some context of what has been on my mind: at some point our ignition packages relies on detecting ign-tools in the system to install the necessary ruby file that enabled the support for it (it would be probably better to install it unconditionally or use a cmake flag to control the installation but we have some legacy from good old times). The cmake check is happening at building time, without ign-tools present in the system you won't get the file installed to support ign command. Same happen for enabling tests that involves ign. This kind of things need to be defined as "Build-deps".

Looking now ign-transport master branch I can see the check for ign-tools but can not see that it is used elsewhere.

Reply:
`ign` support comes from the `conf` and `src/cmd` folders, and I believe these are installed unconditionally

Reply:
so the only reason I see to include ignition-tools in Build-deps is to enable the ign_TEST, which IMHO worth it.

Reply:
👍

Reply:
So... is the current implementation ok?

Reply:
it's not currently a Build depend, I think @j-rivero was suggesting it should be in `Build-Depends` so that a test would be enabled

Reply:
Ok, I put it under the non-dev package's `Build-Depends` because the `-dev` package didn't have that.